### ISSUE TITLE
Flag to add prefix to haxelib commands

### ISSUE THREAD

**Issue by PXshadow** [go2hx/go2hxdoc#4]
github action runner needs to use the lix version of haxe and haxelib so it's necessary to have a flag to enable it by adding ``npx`` prefix to ``haxe`` and ``haxelib`` commands.

**Comment by elliott5** [comment]
@PXshadow : `-prefix` flag added in cd8fc461796fe6ca3416fa77d6ef9a503246fb1b

I'm not sure if that is the issue though.

**Comment by PXshadow** [comment]
@elliott5 Only way is to test commiting to go2hx now.

**Comment by elliott5** [comment]
Not working sadly, as `math/bits` should have passed and been produced.

**Comment by PXshadow** [comment]
> Not working sadly, as `math/bits` should have passed and been produced.

:( I'll use the github code spaces and see if I can fix it that way with some debugging.

**Comment by elliott5** [comment]
Most likely the problem is at 
https://github.com/go2hx/go2hxdoc/blob/cd8fc461796fe6ca3416fa77d6ef9a503246fb1b/rtti/rtti.go#L268

This code selects which directories in the XML file containing RTTI information to pull into the report.

```go
func isThisDir(file, base string, incSubDirs bool) bool {
	dir := filepath.Dir(file)
	if incSubDirs {
		return strings.HasPrefix(dir, base)
	} else {
		return dir == base || dir == base+"_test"
	}
}
```

It should always take the `else` branch. 

Most likely `dir != base`.

When I debugged this issue I printed them out. It produced a LOT of output, but at least I could see the problem.

I suspect something like one of the paths having "//" in it, but not the other ... or something like that.

I'm away on holiday from tomorrow for 10 days, so feel free to debug @PXshadow
